src/test/java/it/pagopa/transactions/configurations/NpgPspApiKeysConfigTest.java

```diff
@@ -191,6 +195,10 @@ void shouldReturnErrorForMissingPspApiKey(NpgClient.PaymentMethod paymentMethod)
                         pspConfigurationJson.formatted(NpgClient.PaymentMethod.SATISPAY),
                         pspToHandle
                 ),
+                npgPspApiKeysConfig.npgSatispayApiKeys(
```

Comment:
is it a duplicate?

Reply:
check code-review

Reply:
good catch, it was a copy-paste typo -> fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/613/commits/9ceb96ea3b420a5b69e60c7f38c1759214600fd5